README.md

```diff
@@ -69,7 +117,7 @@ name = "nativefs"
 
 ### Patch variants
 
-This file contains two patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
+This file contains three patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
```

Comment:
There are 4 types of patch now; also only 2 are described

Reply:
maybe just use bullet points?

Reply:
yeah, I was avoiding updating this paragraph. will do this tomorrow, thanks 👍